CompleteCode/app/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    compileSdkVersion 27
     defaultConfig {
         applicationId "com.rajsuvariya.diffutilsrecyclerview"
         minSdkVersion 16
-        targetSdkVersion 25
+        //noinspection OldTargetApi
+        targetSdkVersion 27
```

Comment:
Please upgrade to 28

Reply:
I thought at least 28 is ok as of now as for 28 compile sdk. Android has recommended the androidx